### ISSUE TITLE
Migration tool from present to future saved query format, 

### ISSUE THREAD

**Issue by larpSTATGL** [statisticssweden/PxWeb#652]
Thank you so much for the frequent sprint reviews. You are all doing a great job.

After the presentation October 7th, I came to think of the info about a migration tool from present to future saved query format.

And then I wonder: is it really necessary? 
Have you considered an alternative strategy where Pxweb itself does a test, whenever a saved query, in the new format fails? Action could then be try to convert from old format, report if still no good and finally overwrite the old format with a new and execute?

Perhaps even a possibility to pass failed queries to the developer team, to help you prioritize.

best regards

**Comment by likp** [comment]
Hi Lars,

Thanks for the valuable feedback.

Our initial idea was in line with your suggestion. We came to the conclusion that I we would go that path then we would have to do this check on every execution of the saved query and that we would still maintain code able to parse, execute and translate current version of saved queries. Since PxWeb 2 and  PxWeb API 2.0 are new version we allow some breaking changes we decided that a migration application was a better alternative.

Best regards
Petros
